src/database/DataTypes/ItemBatch.js

```diff
@@ -30,6 +30,10 @@ export class ItemBatch extends Realm.Object {
     return this.location?.isInBreach() ?? false;
   }
 
+  get breaches() {
```

Comment:
Basically added `breaches` to all the things: from `TransactionItem` to `TransactionBatch` and `StocktakeBatch` `get breaches` is calling `ItemBatch` `get breaches` which is calling `locationMovements get breaches`..